app/api/v2/managers/operation_api_manager.py

```diff
@@ -0,0 +1,23 @@
+from app.api.v2.managers.base_api_manager import BaseApiManager
+from app.api.v2.responses import JsonHttpNotFound, JsonHttpForbidden
+
+
+class OperationApiManager(BaseApiManager):
+    def __init__(self, data_svc, file_svc):
+        super().__init__(data_svc=data_svc, file_svc=file_svc)
+
+    async def get_operation_report(self, operation_id: str, access: dict):
+        operation = await self.get_operation_object(operation_id, access)
+        report = await operation.report(file_svc=self._file_svc, data_svc=self._data_svc)
+
+        return report
+
+    """Object Creation Helpers"""
+    async def get_operation_object(self, operation_id: str, access: dict):
+        try:
+            operation = (await self._data_svc.locate('operations', {'id': operation_id}))[0]
+        except Exception:
```

Comment:
Do we know what exceptions may be caught here? If so, always want to avoid general exception catching (if possible).

Reply:
This has been updated to check for an IndexError, which is returned if the operation is not found